### PR TITLE
Javascript feature

### PR DESCRIPTION
**Pull request by toyman640**
- I added a menu that can be toggled by the hamburger icon
- I made background blur when menu is visible
- I made menu links connect to respective section of page

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @toyman640 and @Mar12358, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn’t have `eslint` setup. Please include `eslint` in your linters and resubmit your project for review.  To learn how to do that use this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Falako, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented:
- Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -51,7 +52,6 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
 - [Live Demo Link](https://toyman640.github.io/Portfolio/)
```
Comment (Zuhaib042) [inline README.md]: - [ ] This `live demo link` is broken. kindly consider removing this link. you can add `Coming soon` here instead. having broken links in our readme makes it unprofessional so kindly consider fixing it.

File: style.css [hunk 2]
```diff
@@ -43,10 +43,63 @@ body {
...
+}
+
+.ham-li {
+  display: flex;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  margin-bottom: 8px;
```
Comment (Zuhaib042) [inline style.css]: - [ ] Kindly consider increasing the margin between the links here and try to make it identical to the figma design as our design should follow the Figma design in `(e.g., font, colors, images, text, margins).` and it is a project requirement.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @toyman640  @rloterh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Nice design
✅ No linters errors
✅ Professional PR
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @toyman640 and @Mar12358 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Use JavaScript to manipulate DOM elements ✔️ 
Use JavaScript events ✔️ 
Made required changes ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**